src/adapterManager.js

```diff
@@ -616,6 +616,12 @@ adapterManager.callBidWonBidder = function(bidder, bid, adUnits) {
   tryCallBidderMethod(bidder, 'onBidWon', bid);
 };
 
+adapterManager.callBidBillableBidder = function(bidder, bid, adUnits) {
+  // Adding user configured params to bidWon event data
+  bid.params = getUserConfiguredParams(adUnits, bid.adUnitCode, bid.bidder);
```

Comment:
I don't think this is necessary. It's already happened in bidWon, and even there I'd question it.